entrypoint/entrypoint/pkg_vuln.py

```diff
@@ -38,6 +39,12 @@ def __init__(self):
         self.cwes = "null"
 
 
+@dataclass
```

Comment:
I have seen several ways of creating data classes in python.
This is the first method I've seen that I like! :)